File: Service/Models/PointSpreadModel.cs
namespace TecmoTourney.Models
{
    public class PointSpreadModel
    {
        public int TournamentId { get; set; }
        public int Player1Id { get; set; }
        public int Player2Id { get; set; }
        public BracketLocation BracketType { get; set; }
        public double Spread { get; set; }
        public int FavoredPlayerId { get; set; }
    }
}

File: Client/tecmo-tourney/src/app/core/models/pointSpread.model.ts
import { BracketLocation } from "src/app/enums";

export interface IPointSpread{
    player1ID: number;
    player2ID: number;
    bracketType: BracketLocation;
    spread: number;
    favoredPlayerId: number;
    summary: string;
    moneyLinePlayer1?: number | null;
    moneyLinePlayer2?: number | null;
    overUnder?: number | null;
}